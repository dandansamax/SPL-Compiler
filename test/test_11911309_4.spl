int test_1_r01(int a, int b)
{
    
    int c[5;

    c[1 = 2;
    c[4+5] =1;
    int;
    c[1+2 =3;
  if (a > b)
  {
    return a;;
  }
  else
  {
    return b;
  }
}