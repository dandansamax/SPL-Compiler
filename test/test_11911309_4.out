Error type A at Line 6: unknown lexeme @
Error type B at Line 6: Missing semicolon ';'
